src/ripple/rpc/handlers/Tx.cpp

```diff
@@ -119,31 +117,31 @@ doTxHelp(RPC::Context& context, TxArgs const& args)
             args.ledgerRange->first, args.ledgerRange->second);
     }
 
-    std::shared_ptr<Transaction> txn;
     auto ec{rpcSUCCESS};
 
-    result.searchedAll = SearchedAll::unknown;
+    using TxPair =
+        std::pair<std::shared_ptr<Transaction>, std::shared_ptr<TxMeta>>;
```

Comment:
Might want to elevate this type alias and use it in more places.

Reply:
Originally we had this alias in `Transaction` namespace, and would use `Transaction::TxPair`. We chose to write out types in the to clearly describe what return types are. 

If you think it's better to use a type alias we can discuss.

Reply:
I don't like this type alias, because there's `Transaction` and there's `STTx` which are both used as datatypes for transactions (as well as `TxMeta` and `STObject` for the metadata). So when I see `Transaction::TxPair`, it's not immediately clear to me what the type of the pair elements are.